### ISSUE TITLE
Contracts endpoint

### ISSUE THREAD

**Issue by smarbos** [adopteitor/adopteitor-api#8]
**Description:**
Create a Contracts Linking an Animal and a Human.  

**Parameters:**
- *humanId (string)*
- *animalId (string)*

**Todo:**
- [x] I think we can use a BEFORE hook to check if both human and animal exists (and do any more check if we need) and then creating the contract. @anabellaspinelli  what do you think?
- [ ] Define contract, maybe this should be a variable set by the shelter.

**Comment by anabellaspinelli** [comment]
Yes about checking existence before persisting. Hoes before broes.

About the "contract" variable:
- is it just a long string for the whole contract? 
- could they have more than one contract template?
- either way, in the admin we could provide a way for them to create one or more contracts with "placeholder" fields for names and dates. I think that would be neat.

**Comment by smarbos** [comment]
- I think a long string for the contract is ok for starting.
- I think one contract template is ok to start. It could be defined in the main configuration of the app.
- I like the placeholder idea, but I think we should keep it simple for the start and add more features in the future.
I think that using placeholders and multiple contracts templates will increase a lot the complexity of this funcionality.

Another think I would love to take care of is the legal aspect of this contract... Again, let's start with something simple and scale it later.
